recipes/sdl/3.x/conanfile.py

```diff
@@ -0,0 +1,193 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import is_apple_os
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, replace_in_file, rm, rmdir, copy
+from conan.tools.gnu import PkgConfigDeps
+from conan.tools.microsoft import is_msvc
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout, CMakeDeps
+from conan.tools.env import Environment
+
+import os
+
+required_conan_version = ">=1.55.0"
+
+# Subsystems, CMakeLists.txt#L234
+_subsystems = [
+    ("audio", []),
+    ("video", []),
+    ("gpu", ["video"]),
+    ("render", ["video"]),
+    ("camera", ["video"]),
+    ("joystick", []),
+    ("haptic", ["joystick"]),
+    ("hidapi", []),
+    ("power", []),
+    ("sensor", []),
+    ("dialog", []),
+]
+
+
+class SDLConan(ConanFile):
+    name = "sdl"
+    description = "A cross-platform development library designed to provide low level access to audio, keyboard, mouse, joystick, and graphics hardware"
+    license = "Zlib"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.libsdl.org"
+    topics = ("sdl3", "audio", "keyboard", "graphics", "opengl")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+
+    options = {
+        **{
+            "shared": [True, False],
+            "fPIC": [True, False],
+        },
+        **{subsystem: [None, True, False] for subsystem, _ in _subsystems}
+    }
+
+    default_options = {
+        **{
+            "shared": False,
+            "fPIC": False,
+        },
+        **{subsystem: None for subsystem, _ in _subsystems}
+    }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+        # If the user has not explicitly set value to any of the subsystems, we should
+        # - If they have deps, its default value is False
+        # - Otherwise, they are True
+        # See CMakeLists.txt#L220, but we #L1167 seems to indicate otherwise, so set them all to True
+        for subsystem, dependencies in _subsystems:
+            # Don't use is with get_safe, it does not return None, but a _PackageOption with a None value
+            # Let the overridden equality check if the option is None
+            if self.options.get_safe(subsystem) == None:
+                default_subsystem_value = True
+                if subsystem == "hidapi" and self.settings.os == "visionOS":
+                    # CMakeLists.txt#L343
```

Comment:
```suggestion
                    # CMakeLists.txt#L378
```

On the 3.2.0 this is line 378